### PR TITLE
The ultimate transparency is here. Yay! =)

### PR DESCRIPTION
**Pull request by techtonik**
This creates a basis for solving #3326. Please add more info.

### CODE REVIEW

**Comment by whit537** [comment]
In #3282 we started calling "ways" "routes."

File: data/routing.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+----------------+------------+--------+------------+----------
+Credit Card     | Balanced   | ?      | At pay day | 
+Bitcoin         | Coinbase   | ?      | ?          |
+
+#### Outs (the way donations moved out of Gratipay)
+
+Way             | Provider   | Fee    | When       | Details
+----------------+------------+--------+------------+----------
+Bank Account    | Balanced   | ?      | At pay day | 
+Paypal          |            | ?      | ?          |
```
Comment (whit537) [inline data/routing.txt]: at payday

File: data/routing.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Bitcoin         |            | ?      | ?          |
```
Comment (whit537) [inline data/routing.txt]: whenever requested via email to support

File: data/routing.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# This data file explains the flow of money through Gratipay
+# and fees taken by different parties.
+
+#### Ins (the way donations are accepted into Gratipay)
+
+Way             | Provider   | Fee    | When       | Details
+----------------+------------+--------+------------+----------
+Credit Card     | Balanced   | ?      | At pay day | 
+Bitcoin         | Coinbase   | ?      | ?          |
```
Comment (whit537) [inline data/routing.txt]: whenever requested via email to support

**Comment by whit537** [comment]
See here for ways, fees, and whens (providers you've already got):

https://gratipay.com/about/faq#how-add-funds
https://gratipay.com/about/faq#how-withdraw-funds

**Comment by techtonik** [comment]
BTW, @whit537, how did you take anchor links from the FAQ page? Do you have to get into HTML markup to get them?

**Comment by whit537** [comment]
Yes. It'd be nice to have those little ¶ links that appear on hover. Feel free to reticket/PR.

**Comment by techtonik** [comment]
@whit537 #3333 Another round number. =)

**Comment by whit537** [comment]
:-)

**Comment by whit537** [comment]
How about we add a table to each of the two relevant FAQs? (I don't see adding a text file in the repo as the way to go.)

**Comment by techtonik** [comment]
@whit537 it is important to note that this is a table that can be parsed by scripts and visualization tools. Although it looks user friendly, it is not a product for end user consumption. This idea behind this is to have a separate `data` layer in our informational architecture (#146).

**Comment by techtonik** [comment]
Adding a parser for the table with inclusion into FAQ just needs a little more time.

**Comment by whit537** [comment]
> This idea behind this is to have a separate data layer in information architecture

@techtonik Hmm ... can you say more about that? What do you mean? (Note that https://github.com/gratipay/inside.gratipay.com/issues/146 is about Inside Gratipay; I've tweaked the ticket title.)

**Comment by techtonik** [comment]
@whit537 the table in data file can be used to provide different visualizations of Gratipay money flow. We can give this table to people and organize a competition to see what is possible to do with it. If the table will be embedded into FAQ page, people will treat it as text.

The information architecture is how the layout of application - where is the data, what components process it, what is the system output. In our case we hardcode the data about fees into the application, in ideal case the fee from the table should be used to:

1. provide value to FAQ
2. pre-calculate exact value when people add funds
3. pre-calculate exact value when people withdraw funds
...
x. access the fee data from external service to monitor when they change

**Comment by techtonik** [comment]
I'd like to clarify the structure of the fees. The information here is not explicit on that Gratipay doesn't take any share of the fee. We need some proof links and explicit statements that these fees are brought in by payment providers and there is nothing extra over in them.

    Balanced  - 2.9% + 30¢
    Coinbase - 1% + 15¢

**Pull request merged**